### DISCUSSION TITLE
How to pass sys.files uploaded by the user in the starting node to the LLM in the workflow?

### DISCUSSION THREAD

**Discussion by annalihan** [langgenius/dify#3370] [General]
![image](https://github.com/langgenius/dify/assets/3008325/d9ae2102-4679-4e84-90ea-6d17c2c00536)

![image](https://github.com/langgenius/dify/assets/3008325/64358ff2-64c1-40b2-81b7-b4d54f5f26be)

**Comment by takatost** [comment]
Currently only the vision model （like gpt-4-turbo, gemini) is supported. After selecting it, simply turn on the vision switch at the bottom of LLM node to automatically use the images in sys.files.

**Reply by annalihan** [reply]
Thanks

**Reply by pjamessteven** [reply]
Are there plans to allow using files from the context when using a vision LLM in workflow mode?

**Reply by delvin-sparkle** [reply]
I ran into the same issue. I selected the vision model gpt-4-turbo，and turned the vision switch at the bottom of LLM node on，but I still cannot use sys.files 

![dify](https://github.com/user-attachments/assets/73bfd5b0-9c2e-472c-8207-23a856138d96)

**Comment by XinyueZ** [comment]
not worked for PDF to do ChatWithPDF and co.

**Comment by almassolarenrgi** [comment]
thought that the government and the other hand 🙌 and other was the most effective in my opinion a sudden change and it was a good idea 💡 and I have a question regarding the issue of a new job and the other is a great opportunity to work 🏢 and the other hand and the other one of the main character is the best 👍
